mozci/push.py

```diff
@@ -394,6 +394,21 @@ def unscheduled_task_labels(self):
         """
         return self.task_labels - self.scheduled_task_labels
 
+    # Because of https://bugzilla.mozilla.org/show_bug.cgi?id=1640758, we can't trust
+    # test-verify when there are no reported failures.
+    # TODO: Drop this function a few months after the bug mentioned above is fixed.
```

Comment:
nit: please move this comment to the docstring and remove the duplicate comments below